libs/shared/console-shared/src/lib/buttons-actions/ui/environment-buttons-actions/environment-buttons-actions.tsx

```diff
@@ -253,6 +253,11 @@ export function EnvironmentButtonsActions(props: EnvironmentButtonsActionsProps)
               contentLeft: <Icon name={IconAwesomeEnum.COPY} className="text-sm text-brand-400" />,
               onClick: () => copyToClipboard(copyContent),
             },
+            {
+              name: 'Export as Terraform',
+              contentLeft: <Icon name={IconAwesomeEnum.FILE_EXPORT} className="text-sm text-brand-400" />,
+              onClick: () => {}, // TODO export as terraform by calling https://api-doc.qovery.com/#tag/Environment-Export/operation/exportEnvironmentConfigurationIntoTerraform
```

Comment:
Hi @RemiBonnet 👋🏽 , is it possible to open a link when clicking this button?

[Here is the documentation](https://api-doc.qovery.com/#tag/Environment-Export/operation/exportEnvironmentConfigurationIntoTerraform).

When we click this button, can we:

1. open a pop-up and explain that the complete environment configuration will be exported as a Terraform manifests
2. Can we add an optional switch to export the secrets? (the user must be an organization admin, otherwise, he will receive a 403 from the API)
3. if the user clicks on "download", a zip file should be prompted for download.

Example of curl command:

```bash
curl -H "Authorization: Bearer $QOVERY_ACCESS_TOKEN" -o ~/Desktop/terraform.zip "https://api.qovery.com/environment/0fc4f959-b197-4b50-8307-6bd3412ba7d0/terraformExport?exportSecrets=false
```

to export the files with the secrets we just need to change the query param for `exportSecrets=true`